pywbem/_server.py

```diff
@@ -266,7 +264,7 @@ def brand(self):
     @property
     def version(self):
         """
-        Version of the WBEM server, as a :term:`string`. `None`, if the version
+        :term:`string`: Version of the WBEM server. `None`, if the version
```

Comment:
Suggestion.  Where do we get the version???

Reply:
We parse it out of the Description property of the CIM_ObjectManager instance, if it contains a pattern `r".+ *Version *([^ ]+)"`